Demo/Common/IntegrationExample.swift

```diff
@@ -74,30 +60,23 @@ internal final class IntegrationExample {
     }
 
     internal func finish(with resultCode: PaymentsResponse.ResultCode) {
-        let success = resultCode == .authorised || resultCode == .received || resultCode == .pending
+        if let finalizableComponent = currentComponent as? FinalizableComponent {
+            let success = resultCode == .authorised || resultCode == .received || resultCode == .pending
+            finalizableComponent.didFinalize(with: success)
```

Comment:
Maybe `DropInComponent.didFinalize()` should also call `stopLoading`?

Reply:
Why not , no harm in it